labs/lab18-helm-charts/README.md

```diff
@@ -0,0 +1,435 @@
+
+```bash
+    $ git init
+    $ git add .
+    $ git commit -m "this is my first commit"
+```
+Now we are ready to create our first release:
+
+```bash
+    $ replicated release create --auto --promote lab18-helm-charts -y
+```
+
+## Deploy the Application
+
+We now have something we can deploy! As part of this lab, you were provided with the IP address of a Virtual Machine. Log in using `ssh kots@{IP-Address}` using the IP address provided. The password will be provided during this lab.
+
+Once you are on the terminal, copy the "Embedded Cluster" install command from the channel (`lab18-helm-charts`). It should look like
+```
+curl -sSL https://k8s.kurl.sh/YOUR-APP-NAME-lab18-helm-charts | sudo bash
+```
+
+Go through the installation steps in kotsadm to install the application. You should have already completed [Lab0: Hello World](https://github.com/replicatedhq/kots-field-labs/blob/main/labs/lab00-hello-world/README.md), so this should be familiar.
+
+## Mapping Field Values
+
+Let's take some of the field values and map them to the ui!
+
+We are going to make two changes:
+* Map the `wordpressBlogName` field in the Values.yaml file to a UI field.
+* Override the default port of 80 to 8080.
+
+### Copy the field name from the Values file
+
+For this lab we are going to use the `wordpressBlogName` field in the values file and expose it in the UI. For reference, we are going to be referencing fields in the ../charts/bitnami/wordpress/values.yaml file.
+
+![Values File](./img/values-file.png)
+
+### Create a field in the Config.yaml file
+
+Create a file under `manifests` named `kots-config.yaml` file with the following contents:
+
+```yaml
+apiVersion: kots.io/v1beta1
+kind: Config
+metadata:
+  name: wordpress-config
+spec:
+  groups:
+    - name: wordpress
+      title: Wordpress
+      description: Wordpress Defaults
+      items:
+        - name: wordpressBlogName
+          title: Wordpress Blog Name
+          type: text
+
+```
+
+As you can see, we are using the values field name as the config field name and used for the title, which is what the end user will see.
+
+
+### Map the Config file field with the field in the Values file
+
+Edit the `wordpress.yaml` file and scroll down to the values section and make the following changes:
+
+```diff
+# values are used in the customer environment, as a pre-render step
+# these values will be supplied to helm template
+-- values: {}
+++ values:
+++   wordpressBlogName: '{{repl ConfigOption "wordpressBlogName"}}'
+```
+
+Now we are ready to create our next release:
+
+```bash
+    $ replicated release create --auto --promote lab18-helm-charts -y
+```
+
+## Update the Application
+
+Let's see our changes on the Admin Console. Click on Check for Update as shown below.
+
+Before you click on Deploy, note that now you have a Config tab with our field in it.
```

Comment:
```suggestion
```

I got a weird error when I navigated to the config tab since sequence 1 release doesn't have a Config.yaml, you get this weird issue:  

<img width="381" alt="Screen Shot 2022-07-10 at 2 44 16 PM" src="https://user-images.githubusercontent.com/3730605/178160014-9e76abe5-341c-4131-ac0b-73901981e964.png">

We should consider either including an empty config in the initial release, or we could instruct the user to deploy before navigating to Config.